internal/agentdeployer/_static/dockerfile.tmpl

```diff
@@ -14,9 +14,10 @@ USER root
 
 {{ if ne $provisioning_script_contents "" }}
 COPY {{ $provisioning_script_filename }} .
-RUN chmod u+x {{ $provisioning_script_filename }} && \
+RUN set -eu ; \
```

Comment:
Even without this it should fail in some cases, right?

Reply:
I added this to be sure and force that it would fail the RUN step.

Reply:
I'll do some tests without the `set -eu` in this RUN step.

Reply:
As this is just a one line, it looks like it is not needed to add `set -eu`.
The problem would be if this is changed at some point to be a multiline script like:
```
RUN chmod u+x {{ $provisioning_script_filename }} ; \
  {{ $provisioning_script_language }} ./{{ $provisioning_script_filename }} ; \\
  rm {{ $provisioning_script_filename }}  
```